### PR TITLE
doc: release notes for 12.2.3

### PR DESCRIPTION
**Pull request by theanalyst**
Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

### CODE REVIEW

File: doc/release-notes.rst [hunk 1]
```diff
@@ -2,6 +2,245 @@
  Release Notes
 ===============
 
+v12.2.3 Luminous
+================
+
+This is the second bugfix release of Luminous v12.2.x long term stable release
```
Comment (liewegas) [inline doc/release-notes.rst]: third?

File: doc/release-notes.rst [hunk 2]
```diff
@@ -2,6 +2,245 @@
...
+series. It contains a range of bug fixes and a few features across Bluestore,
+CephFS, RBD & RGW. We recommend all the users of 12.2.x series update.
+
+For more detailed information, see :download:`the complete changelog <changelog/v12.2.2.txt>`.
```
Comment (liewegas) [inline doc/release-notes.rst]: s/12.2.2/12.2.3/

File: doc/release-notes.rst [hunk 3]
```diff
@@ -2,6 +2,245 @@
...
+
+
+Notable Changes
+---------------
+
+- *CephFS*:
+
+  * The CephFS client now catches failures to clear dentries during startup
+    and refuses to start as consistency and untrimmable cache issues may
```
Comment (liewegas) [inline doc/release-notes.rst]: The CephFS client now checks for older kernels' inability to reliably clear dentries from the kernel dentry cache

File: doc/release-notes.rst (hunk 1 above)
Comment (theanalyst) [reply doc/release-notes.rst]: sorry copy-paste errors, fixing

**Comment by theanalyst** [comment]
jenkins test docs

**Comment by theanalyst** [comment]
jenkins render docs

**Pull request merged**